src/executor/evaluate/mod.rs

```diff
@@ -228,6 +228,14 @@ async fn evaluate_function<'a, T: 'static + Debug>(
         }
     };
 
+    let eval_to_f64 = |name: &'static str, expr| async move {
+        match eval(expr).await?.try_into()? {
+            Value::F64(number) => Ok(number),
+            Value::I64(number) => Ok(number as f64),
+            _ => Err::<_, Error>(EvaluateError::FunctionRequiresF64Value(name.to_owned()).into()),
```

Comment:
1. Is this `GCD` only works for integer values...?
We may not even need to accept `f64` in here.

2. Let's get `Value::Null`
It is sad but SQL language supports `NULL` value so...
Usual behavior is `NULL` comes into functions params, then result also becomes `NULL`.